src/templates/program-multiple.js

```diff
@@ -66,3 +67,9 @@ const ProgramsPage = ({ data }) => {
 
 export default ProgramsPage;
 
+
+// TODO
```

Comment:
@Nikhil-Ladha we can't access activeOption here. so how do we pass <SEO title={programs[activeOption].frontmatter.program} /> ?

and is it the right practise to have different title for same url ?

Reply:
Technically, we are doing nothing wrong here.
The path changes for different programs, it is just that for other years of the same program we use the same title. That is totally fine, as for SEO benefits we are generating individual pages as well.

Coming to the access part, create a global variable `let activeTitle = ""` and then update its value inside the `ProgramsPage` component to be `activeTitle=programs[activeOption]`. I think this should work.

Reply:
@Nikhil-Ladha  not working . the title does is not getting updated it remains the default one

Reply:
in the mean time i have used {programs[0].frontmatter.program} which would display correct program name in title

<img width="1440" alt="Screenshot 2023-01-14 at 5 46 00 PM" src="https://user-images.githubusercontent.com/53316345/212471203-caea3ae9-bb15-4486-b891-bfe5120ca418.png">

Reply:
This works as well.